docs/howto/db.md

```diff
@@ -0,0 +1,29 @@
+(db)=
+# How to use a SQLair database
+To create a SQLair database and run SQLair queries on it, you need to wrap a
+`sql.DB` database object from the `database/sql` package. SQLair is a
+convenience wrapper around the database, it does not create the DB itself.
+Operations such as opening, closing and managing the DB settings is done through
+`database/sql`.
+
+## Wrap a database with SQLair
+To wrap a `sql.DB` with SQLair, use the `sqlair.NewDB` method. This will return
+a `sqlair.DB`.
+
+> See more:
+[`sqlair.NewDB`](https://pkg.go.dev/github.com/canonical/sqlair#NewDB),
+[`sqlair.DB`](https://pkg.go.dev/github.com/canonical/sqlair#DB),
+[`database/sql.DB`](https://pkg.go.dev/database/sql#DB)
+
+## Query a SQLair database
+
+See {ref}`query` and {ref}`tx`.
+
+## Unwrap a SQLair database
+
+To unwrap a SQLair database and get out the `sql.DB`, use `DB.PlainDB`. SQLair
+does not handle closing the database, this should be done through the
+`database/sql` package.
+
+> See more:
+[`sqlair.PlainDB`](https://pkg.go.dev/github.com/canonical/sqlair#DB.PlainDB),
```

Comment:
```suggestion
[`sqlair.PlainDB`](https://pkg.go.dev/github.com/canonical/sqlair#PlainDB)
```

Reply:
Ah, this is the other side that should change, sqlair.PlainDB -> DB.PlainDB